src/main/java/io/confluent/copycat/hdfs/AvroRecordWriterProvider.java

```diff
@@ -41,7 +41,10 @@
 
     final Schema schema = record.valueSchema();
     final FSDataOutputStream out = path.getFileSystem(conf).create(path);
-    writer.create(avroData.fromCopycatSchema(schema), out);
+    org.apache.avro.Schema avroSchema = avroData.fromCopycatSchema(schema);
+    int version = schema.version();
+    avroSchema.addProp("version", String.valueOf(version));
```

Comment:
The `AvroData` class should already be saving the version information to a property called `copycat.version`. You could probably just reuse that instead.